backend/src/constants/ErrorMessage.ts

```diff
@@ -7,4 +7,5 @@ export default {
     SOMETHING_WENT_WRONG: 'Something went wrong',
```

Comment:
I did provide specific error messages for known errors but this is for catch block in case something goes wrong on transaction submission or invalid balance due to high gas price etc

Reply:
In this scenario never leave the user without a course of remediation, it looks really bad on us. At least tell them what to do next